tokens/governance/proposals.md

```diff
@@ -32,51 +29,65 @@ Moonbeam的治理系统正在更新中。下一个阶段的治理称为OpenGov (
 
 --8<-- 'text/governance/lead-in-definitions.md'
 
-请确保您查看每个网络和track的[治理参数](/learn/features/governance/#governance-parameters-v2)。
+请确保您查看每个网络和track的[治理参数](/learn/features/governance/#governance-parameters-v2){target=_blank}。
 
 ## 提案步骤 {: #roadmap-of-a-proposal }
 
 本教程将涵盖提案步骤图的前几个步骤，如下图突出显示的步骤所示。您将学习如何提交提案想法至[Moonbeam社区论坛](https://forum.moonbeam.foundation/){target=_blank}、提交原像以及使用原像哈希在链上提交提案。
 
-您可以在[治理概览页面的OpenGov (Governance v2)提案步骤](/learn/features/governance/#roadmap-of-a-proposal-v2){target=_blank}部分找到详细的解释。
+您可以在[治理概览页面的OpenGov提案步骤](/learn/features/governance/#roadmap-of-a-proposal-v2){target=_blank}部分找到详细的解释。
+
+![Proposal Roadmap](/images/tokens/governance/proposals/proposal-roadmap.png)
+
+## 提交您的想法至论坛 {: #submitting-your-idea-to-the-forum }
+
+在开始操作提交提案的步骤之前，您需要先熟悉[Moonbeam社区论坛](https://forum.moonbeam.foundation/){target=_blank}平台。强烈建议您先在论坛上发布任何提案相关的帖子以接收社区反馈。在继续提交原像和提案之前，您需要预留5天时间供社区讨论并在Moonbeam社区论坛帖子上提供反馈。
+
+要访问Moonbeam社区论坛，您必须是[Moonbeam Discord](https://discord.com/invite/PfpUATX){target=_blank}的社区成员之一。然后，您可以使用您的Discord注册以访问论坛。
+
+登陆后，您可以查看最新讨论、加入对话，并为您拥有的提案想法创建自己的讨论。在首次发布帖子或发表评论前，请确保您已熟悉[常见问答](https://forum.moonbeam.foundation/faq){target=_blank}并了解社区准则。
+
+![Moonbeam Forum Home](/images/tokens/governance/treasury-proposals/treasury-proposal-1.png)
+
+如果您已准备好发布提案详情的帖子，您可以前往**Governance**页面并点击**Democracy Proposals**。
+
+![Governance page on Moonbeam Forum](/images/tokens/governance/proposals/proposals-1.png)
```

Comment:
```suggestion
![Governance page on Moonbeam Forum](/images/tokens/governance/proposals/v1/proposals-1.png)
```

Reply:
this was changed in corresponding PR https://github.com/moonbeam-foundation/moonbeam-docs/pull/718/files#diff-cf355dd387929cb62b8e4ff27f3909270b4fa88b3b12b9c8dd82ce9c0eb3706c